Update final_report.md

defabdullah opened the pull request:
I added my individual contribution report lately.

defabdullah committed:
Update final_report.md

Simurgan committed:
add progress summary ömer şükrü

Simurgan commented:
I only added my progress summary actually, since I forgot to add it to the report yesterday.
The rest of the changes are caused by autoformatter of my markdown extension.

defabdullah committed:
Update final_report.md

Add my part at the progress based on teamwork also add deleted part which i add at the first commit

laylaylo committed:
add progress summary for leyla

laylaylo committed:
addded deleted part
